### DISCUSSION TITLE
Welcome to channels Discussions!

### DISCUSSION THREAD

**Discussion by carltongibson** [django/channels#1493] [General]
## 👋 Welcome!

  We’re using Discussions as a place for those questions that aren't really issues, or aren't really issues yet:

  * Deployment issues. 
  * Usage questions.
  * Ideas and suggestions.

Please be welcoming to others and open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zhaoxun** [comment]
Hello! Our team uses Django + Websocket to write a server. I noticed it employs channels to manage websocket connections.
